mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java

```diff
@@ -0,0 +1,162 @@
+package org.mifos.mobilewallet.mifospay.faq.ui;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ExpandableListView;
+import android.widget.Toast;
+
+import org.mifos.mobilewallet.mifospay.R;
+import org.mifos.mobilewallet.mifospay.base.BaseActivity;
+import org.mifos.mobilewallet.mifospay.faq.FAQContract;
+import org.mifos.mobilewallet.mifospay.faq.FAQListAdapter;
+import org.mifos.mobilewallet.mifospay.faq.presenter.FAQPresenter;
+import org.mifos.mobilewallet.mifospay.utils.Constants;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import javax.inject.Inject;
+
+
+public class FAQActivity extends BaseActivity implements FAQContract.FAQView {
+
+    private ExpandableListView expandableListView;
+
+    private FAQListAdapter faqListAdapter;
+
+    private List<String> listDataGroup;
+
+    private HashMap<String, List<String>> listDataChild;
+
+    @Inject
+    FAQPresenter mPresenter;
+    FAQContract.FAQPresenter mFAQPresenter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_faq);
+
+        showBackButton();
+        setToolbarTitle(Constants.FAQ);
+
+        // initializing the views
+        initViews();
+
+        // initializing the listeners
+        initListeners();
+
+        // initializing the objects
+        initObjects();
+
+        // preparing list data
+        initListData();
+    }
+
+    @Override
+    public void setPresenter(FAQContract.FAQPresenter presenter) {
+        mFAQPresenter = presenter;
+    }
+
+    /**
+     * method to initialize the views
+     */
+    public void initViews() {
+
+        expandableListView = findViewById(R.id.faq_list);
+
+    }
+
+    /**
+     * method to initialize the listeners
+     */
+    public void initListeners() {
+
+        // ExpandableListView Group expanded listener
+        expandableListView.setOnGroupExpandListener(new ExpandableListView.OnGroupExpandListener() {
+
+            @Override
+            public void onGroupExpand(int groupPosition) {
+                Toast.makeText(getApplicationContext(),
+                        listDataGroup.get(groupPosition) + "Collapsed",
+                        Toast.LENGTH_SHORT).show();
+            }
+        });
+
+        // ExpandableListView Group collapsed listener
+        expandableListView.setOnGroupCollapseListener(new ExpandableListView.OnGroupCollapseListener() {
```

Comment:
This line is has more than 100 characters, just fix this and it should work.

Reply:
Will do it in a few minutes.

But the gradlew check i've performed in my computer reported errors in the master branch, how will this fix that?

Reply:
Ignore that as of now, focus on this PR. After you get your PR merged, delete your fork and fork again and it should work.